frontend/src/utils/enum.ts

```diff
@@ -0,0 +1,14 @@
+export const EnumPriceSort = {
```

Comment:
please rename this: this isn't a real enum. or, make it a real enum with typescript. also, the file should be something more descriptive than `enum.ts`.

Reply:
Read and think enum not much help project so I reverted it. Thank you